lib/BackgroundJob/ExpirePageTrash.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
```

Comment:
Good point about the `strict_types` directive. I will add it to all missing PHP files in a follow-up PR. About the licence headers, we decided somewhere in the past that it's a bit redundant to add them to each and every file and instead just stick to the global `COPYING` file. Also, because the copyright attribution in the headers is a bit debatable due to the fact that often only the initial authors are listed. But I don't have problems to rethink this practice if there's good reasons to do so.

Reply:
I am only aware the license headers were always added. If it was decided here not to include them and it is legally sufficient, it's all fine with me.